src/components/accordion/accordion.spec.js

```diff
@@ -52,6 +52,7 @@ describe('Accordion', () => {
     ).mockImplementation(() => contentHeight);
   };
 
+  testStyledSystemSpacing(props => <Accordion { ...props } />, () => wrapper.find(StyledAccordionContainer));
```

Comment:
```suggestion
  testStyledSystemSpacing(props => <Accordion { ...props } title='foo'  />, {}, () => wrapper.find(StyledAccordionContainer));
```

You are passing the `wrapper.find(StyledAccordionContainer)` in as the `defaults` here so need to add the `{}`, `title` is also a required prop in `Accordion`.

Reply:
Actually that isn't true, the `wrapper.find(StyledAccordionContainer)` isn't the defaults, it's the `styleContainer` parameter - the second thing you pass into this function needs to be either `{}` or `undefined` and the above should be the third thing passed in.

Reply:
Apologies, that's what I meant, he is passing the `wrapper.find(StyledAccordionContainer)` as the second param so it is being read as the `defaults` rather than the styleContainer

Reply:
I actually misread your comment there as well sorry, we're both saying the same thing but I thought you'd said the opposite